cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/goals/MinTopicLeadersPerBrokerGoal.java

```diff
@@ -51,7 +52,7 @@
 public class MinTopicLeadersPerBrokerGoal extends AbstractGoal {
   private static final Logger LOG = LoggerFactory.getLogger(MinTopicLeadersPerBrokerGoal.class);
   private final String _replicaSortName = replicaSortName(this, true, false);
-  private Set<String> _mustHaveTopicLeadersPerBroker;
+  private Map<String, Integer> _mustHaveTopicMinLeadersPerBroker;
```

Comment:
Is it possible to add some comments on when the value of this map is relevant -- i.e. only when `min.topic.leaders.per.broker` is non-positive -- and what it is used for?

Reply:
Good call, added an inline comment

Reply:
To clarify, this `_mustHaveTopicMinLeadersPerBroker` is always used to retrieve the min leaders requested for each topic. When `min.topic.leaders.per.broker` is set to `0` the computed value is stored as key, when >0 the specified `min.topic.leaders.per.broker` value is stored for each topic in the map.
This has been done to keep the changes at minimum and replace the usage of `_mustHaveTopicLeadersPerBroker` which was a `Set` with the new `_mustHaveTopicMinLeadersPerBroker` `Map`